inline.go

```diff
@@ -738,12 +784,6 @@ func (p *parser) inlineHtmlComment(out *bytes.Buffer, data []byte) int {
 
 // '<' when tags or autolinks are allowed
 func leftAngle(p *parser, out *bytes.Buffer, data []byte, offset int) int {
-	if p.flags&EXTENSION_INCLUDE != 0 {
```

Comment:
It depends how you look at it. This is the place where code-blocks get included in the second-pass. We do it in the first-pass hence this is not necessary any more.